recipes/recipes_emscripten/python/recipe.yaml

```diff
@@ -1,40 +1,30 @@
 context:
   version: 3.11.3
-  pyodide_version: 0.24.0
   name: python
 package:
   name: ${{name}}
   version: ${{ version }}
 
 source:
-- url: https://github.com/pyodide/pyodide/archive/refs/tags/${{pyodide_version}}.tar.gz
-  sha256: c8350c4780be44fe763a66f5024197025b0f8740cc4a7c9ad833a89c69916276
+- url: https://www.python.org/ftp/python/${{ version }}/Python-${{ version }}.tar.xz
+  sha256: 8a5db99c961a7ecf27c75956189c9602c968751f11dbeae2b900dbff1c085b5e
 
 build:
-  number: 24
-  string: h_hash_24_cpython
+  number: 0
```

Comment:
we need to bump the existing bld nummer since its the same version (ie has to be 25)